cli/src/commands/auth.rs

```diff
@@ -0,0 +1,45 @@
+use crate::commands::Command;
+use crate::config::CliConfig;
+use crate::errors::{ExitCode, Fallible};
+use crate::style::KEY;
+use crate::telemetry::Session;
+use crate::terminal::sensitive;
+use console::style;
+use log::{info, warn};
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+#[structopt(rename_all = "kebab-case")]
+pub enum Auth {
+    /// Setup your auth stuff
```

Comment:
does this part print out when you run `help`? If so, can you remove it? `auth stuff` doesn't give much help in what the command does 😅

Reply:
Ha, good catch, missed that.

Reply:
![stuff](https://media.giphy.com/media/11sClUt4JRNrZC/giphy.gif)